notebooks/06_Serving/06_02_Kfserving.ipynb

```diff
@@ -0,0 +1,493 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for KFServing SDK \n",
```

Comment:
... Once you combine two files. I would suggest you to have another header like `KFServing Sample` or something more meaningful. Otherwise, people see ` Sample for KFServing SDK` first and they think this notebook is just for SDK

Reply:
It makes sense, fixed.